.github/CODEOWNERS

```diff
@@ -1,4 +1,4 @@
 # Note: later rules override earlier rules.
 
 # Default
-* @IntersectMBO/ouroboros-consensus @IntersectMBO/consensus-superowners
+* @IntersectMBO/ouroboros-consensus-maintainers @IntersectMBO/consensus-superowners
```

Comment:
Could even remove `@IntersectMBO/consensus-superowners` as it doesn't seem to exist:

>  Unknown owner on line 4: make sure the team @IntersectMBO/consensus-superowners exists, is publicly visible, and has write access to the repository

Reply:
Thanks to @jasagredo, I learned that this group might be revived in the future, so let's leave it there for now.